assets/scss/components/foundation/dropdown/_dropdown.scss

```diff
@@ -18,6 +18,11 @@
     }
 }
 
+.dropdown-menu.is-open {
```

Comment:
I'm living as is because if I do this:
```scss
.dropdown-menu
    &.is-open {
         display: block;
    }
}

the scss linter starts whining with
```
  21: [W] MergeableSelector:  Merge rule `.dropdown-menu` with rule on line 12
```
And if I merge it with L2 I'll have to add one more nested level and disable `NestingDepth` in scss-lint

I don't like having a limit of 2 on `NestingDepth`. Any reasons for that?